### ISSUE TITLE
Switch to disable JIT altogether?

### ISSUE THREAD

**Issue by ArKay74** [michalsc/Emu68#202]
Would it be possible to add an interpreted mode where it doesn't use the JIT at all? Would be interesting to see how fast it is without it. 

Most emulators offer a JIT Compiler switch,

**Comment by michalsc** [comment]
Yes, it would be possible but it is something for the future (the interpreter has to be written from scratch)
